internal/oc/release.go

```diff
@@ -164,7 +188,20 @@ func (r *release) GetReleaseArchitecture(log logrus.FieldLogger, releaseImage st
 	if image == "" {
 		return nil, errors.New("no releaseImage nor releaseImageMirror provided")
 	}
-	cmd := fmt.Sprintf(templateImageInfo, image)
+
+	icspFile, err := r.getIcspFileFromRegistriesConfig(log)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create file ICSP file from registries config")
+	}
+	defer removeIcspFile(icspFile)
+
+	var cmd string
+	if icspFile == "" {
+		cmd = fmt.Sprintf(templateImageInfo, image)
+	} else {
+		cmd = fmt.Sprintf(templateImageInfoWithIcsp, image, icspFile)
```

Comment:
@bfournie Maybe change to `icspFile, image` here?

Reply:
I just tested and it worked with
`cmd = fmt.Sprintf(templateImageInfoWithIcsp, image, icspFile)`

Reply:
Oof, thanks @chadcrum, nice catch!  I've fixed it, PR is updated.